planetiler-core/src/main/java/com/onthegomap/planetiler/ForwardingProfile.java

```diff
@@ -55,9 +55,13 @@ public abstract class ForwardingProfile implements Profile {
   @SuppressWarnings("java:S3077")
   private volatile MultiExpression.Index<FeatureProcessor> indexedSourceElementProcessors = null;
 
-  protected ForwardingProfile(PlanetilerConfig config, Handler... handlers) {
+  public ForwardingProfile(PlanetilerConfig config) {
```

Comment:
This should just be protected since only a subclass can ever call it.

Reply:
Thanks, adjusted.